platform/android/java/editor/src/release_debug/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<string name="godot_editor_name_string">Godot Editor (release-debug)</string>
```

Comment:
It's probably best not to introduce another naming discrepancy here:
```suggestion
	<string name="godot_editor_name_string">Godot Editor (release_debug)</string>
```

What's the difference between this and the one in `editor/src/main` though?

Reply:
Since the `release` target is not valid for the editor build, this is the same as `editor/src/main` so I removed the `release_debug` subdirectory (including this file).

Reply:
The resulting naming convention is `Godot Editor` for the `release_debug` builds and `Godot Editor (debug)` for the `debug` builds.